<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWidget</class>
 <widget class="QWidget" name="CMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PurrFX drum machine</string>
  </property>
  <layout class="QHBoxLayout" name="layMain" stretch="1,3">
   <item>
    <layout class="QVBoxLayout" name="laySamples">
     <item>
      <widget class="QPushButton" name="btnLoad">
       <property name="text">
        <string>Load sample</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="lstSamples"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layOptionsControls">
     <item>
      <layout class="QHBoxLayout" name="layControls">
       <item>
        <widget class="QPushButton" name="btnPlay">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPause">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStop">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lblFPS">
         <property name="text">
          <string>FPS:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spbFPS">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="edtSequences">
       <property name="font">
        <font>
         <family>Courier</family>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
